_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Team offsite meeting: workshops, focus, and fun"
+date: "2016-07-04 13:37:42"
+image: blog-header/ep6offsite2016_back.jpg
+categories: events
+authors: ["Sandra", "Oliver"]
+---
+
+Mid June, the epages 6 team went on a two-day offsite meeting -- in Jena! How is it an offsite if you stay in the city you work in? Well, due to the size of the team and various team members having family obligations it was difficult to find a date and a location for this offsite.
+Therefore, we agreed to stay in Jena. To avoid the usual office interruptions we booked the [Creative Room of the Volkshochschule Jena](https://www.vhs-jena.de/de/service/raumvermietung/649482#/collapse4) for two days. Additionally, we organised a cool team event in the forest area in Camburg. But let us tackle the serious issues first!
+
+## Team constellation and focus
+
+Just recently, our team was restructured. New team members were added, whereas some colleagues moved to other teams. Doing a team offsite was just the right setting to get to know each other better, and boost the team spirit. As we are facing new challenges, one big topic was focusing on our new tasks.
+
+## Workshops
+
+{% image blog/blog-team-ep6.jpg 50% right %}
+
+In order to spread knowledge and improve our [T-shaped skills](https://en.wikipedia.org/wiki/T-shaped_skills) we prepared various workshops.
+In advance we identified topics that are relevant to everyone in the team and topics that are only relevant to a certain guild.
+To make best use of the limited time we held some workshops on parallel tracks.
+
+Of interest to everyone was the "JavaScript in epages 6" session chaired by Markus. He explained in detail how different JavaScript frameworks are integrated in epages 6 and what to consider when writing JavaScript in this environment. Using live coding he demonstrated how the template engine, localization and JavaScript compression work.
+
+{% image blog/blog-ep6-frontend-workshop.jpg 50% left %}
+
+Oliver explained selected [OWASP threats](https://www.owasp.org/index.php/Cheat_Sheets). Afterwards we tried to exploit our software. While we identified some problems, we also concluded that epages 6 is pretty resilient against the most common attacks: injections and cross-site scripting.
+
+Peter outlined our current development pipeline process used primarily for QA and collaboration on more complex features.
+Additionally he clarified the different build  and configuration setups on our team VMs.
```

Comment:
Additionally he clarified the different build  and configuration setups on our team VMs. > Additionally, he clarified the different build and configuration setups on our team VMs. (comma after additionally and remove white space after "build")